### DISCUSSION TITLE
Unitrunker/Trunking Recorder help

### DISCUSSION THREAD

**Discussion by JPhillips21366** [chuot/rdio-scanner#271] [Q&A]
I am new to rdio-scanner and received information about it through the Google Unitrunker group.  Almost finished setting up.  

Not familiar with either apikeys or dirwatch.  

I selected Dirwatch with the following setting - Disabled is set to off, delete after is set to on, Type - Trunk Recorder which I believe is the same as Trunking Recorder.  Path to the CaptureDir - I copied the Trunking Recorder Output Folder that is under the General tab of Trunking Recorder - C:\Users\HP\Documents\Trunking Recorder.  It doesn't like that and so I can't save it.

It also refers to a jdon file and I'm not familiar with that one either.

I'm pretty good with Unitrunker and Trunking Recorder, but this one has me stumped.  

Thank you in advance for your help.

John

Guess everyone can disregard.  Someone from the Google Unitrunker group just informed me that Trunk and Trunking Recorder are 2 different programs.

**Comment by bobfeser** [comment]
Did you ever figure out the setup for this? I am trying to do a setup but having issues with DIRWATCH and SYSTEM. I put in my system id and added all the TG and it keeps broadcasting everything on the one TG that I had to select in DIRWATCH. 
Did you have to make a DIRWATCH for every TG?

any help would be great.

Bob
